### ISSUE TITLE
KDM Release - k8s patches for Nov

### ISSUE THREAD

**Issue by maggieliu** [rancher/rancher#29981]
Add new k8s patches
1.19.4 - 2.5.0+
1.18.12 - 2.4.5+
1.17.14 - 2.4.4+

**Comment by superseb** [comment]
@maggieliu v1.18.10 is v2.4.5+, see https://github.com/rancher/kontainer-driver-metadata/blob/release-v2.5/rke/k8s_version_info.go#L318. Can you change it or let me know if this still needs to be different for v1.18.11

**Comment by maggieliu** [comment]
@superseb Updated the 1.18.11 to be 2.4.5+

**Comment by sowmyav27** [comment]
RKE validations using the following RKE versions and applicable k8s versions by deploying clusters, running basic validation checks works fine:
```
v1.1.12-rc1
- 1.18.12-rancher1-1
- 1.17.14-rancher1-1

v1.2.3-rc1
- 1.19.4-rancher1-1
- 1.18.12-rancehr1-1
- 1.17.14-rancher1-1
```

**Comment by jiaqiluo** [comment]
the following new versions are validated on rancher:v2.5.2 
- 1.19.4-rancher1-1
- 1.18.12-rancehr1-1
- 1.17.14-rancher1-1

The following scenarios are validated:
- provision a cluster with an older minor version, then upgrade to the new version 
- provision a cluster with the new version

**Comment by aaronRancher** [comment]
**Fresh install checks**
Network checks on the following Rancher setups and k8s versions pass.
```
Rancher v2.5.1
k8s versions:
- v1.19.4-rancher1-1
- v1.18.12-rancher1-1
- v1.17.14-rancher1-1

Rancher v2.4.5
k8s versions:
- v1.18.12-rancher1-1
- v1.17.14-rancher1-1

Rancher v2.4.4
k8s versions:
- v1.17.14-rancher1-1
```

**Comment by superseb** [comment]
* KDM PRs merged to `release-v2.4` and `release-v2.5`
* RKE v1.1.12 + v1.2.3 released
* Forum: https://forums.rancher.com/t/kubernetes-v1-17-14-v1-18-12-v1-19-4/
